### PR TITLE
Adding php-cache's adapter-bundle and cache-bundle

### PR DESCRIPTION
**Pull request by aequasi**
| Q             | A
| ------------- | ---
| License       | MIT

@Nyholm

### CODE REVIEW

File: cache/cache-bundle/1.0/config/packages/cache.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    validation:
+        enabled: "%env(bool:VALIDATION_CACHE_ENABLED)%"
+        service_id: "cache.provider.default"
+        use_tagging: true
+
+framework:
+    annotations:
+        cache: 'cache.service.annotation'
+    validation:
+        cache: 'cache.service.validation'
```
Comment (stof) [inline cache/cache-bundle/1.0/config/packages/cache.yaml]: I'm not sure silently replacing the Symfony builtin cache is good.

Comment (aequasi) [reply cache/cache-bundle/1.0/config/packages/cache.yaml]: How would you suggest we replace it?

**Comment by aequasi** [comment]
@stof had a talk with @nicolas-grekas about cache-bundle. We are going to actually remove the validator, annotation, and serializer caches from `cache-bundle`

**Comment by fabpot** [comment]
@symfony-flex-server review please

**Pull request merged**